test/integration/kubernetes_deploy_test.rb

```diff
@@ -927,4 +927,21 @@ def test_resource_watcher_raises_after_timeout_seconds
       "If you expected it to take longer than 5 seconds for your deploy to roll out, increase --max-watch-seconds."
     ], in_order: true)
   end
+
+  def test_adds_namespace_labels_to_statsd_tags
+    desired_tags = %W(context:#{KubeclientHelper::MINIKUBE_CONTEXT} namespace:#{@namespace} foo:bar)
+    hello_cloud = FixtureSetAssertions::HelloCloud.new(@namespace)
+    kubeclient.patch_namespace(hello_cloud.namespace, metadata: { labels: { foo: 'bar' } })
+    metrics = capture_statsd_calls do
+      assert_deploy_success(deploy_fixtures("hello-cloud"))
```

Comment:
this fixture set is HUGE so don't deploy the whole thing unless you need to. You can select specific files from it using the `subset:` arg. If you don't need anything specific, I'd suggest `"configmap-data.yml"`, which will deploy instantly.

Reply:
Yea we just want to check that it is grabbing the labels from the namespace, we don't care about what it is deploying. Thanks :)